test/system/005-info.bats

```diff
@@ -53,6 +53,21 @@ store.imageStore.number   | 1
 
 }
 
+# 2021-04-06 discussed in watercooler: RHEL must never use crun, even if
+# using cgroups v2.
+@test "podman info - RHEL must use runc" {
```

Comment:
Can you make this only for RHEL < 9.0?

Reply:
Done. New changes pushed, with a FIXME because I have no idea what to expect on CentOS. (Since we don't run system tests on it, I don't expect this to ever trigger, but if anyone reading this knows what `/etc/os-release` looks like on CentOS, I'd appreciate seeing `$VERSION_ID`, TIA)